aeronautical/asterix/cat_034_catalog.ksy

```diff
@@ -0,0 +1,52 @@
+meta:
+  id: cat_034_catalog
+  license: GPL-3.0-only
+  endian: be
+  imports:
+#    - bit_map
+    - fixed
+    - explicit
+    - extended
+    - repetitive
+    - compound
+  ks-version: 0.9
+
+doc: |
+  Implementantion of ASTERIX format.
+
+  From Eurocontrol Asterix web page at
+  https://www.eurocontrol.int/asterix
+
+  Category 034 Catalog definition file
+
+  ASTERIX Part 2b Category 34
+  Transmission of Monoradar Service Messages
+
+  SUR.ET1.ST05.2000-STD-02b-01
```

Comment:
This actually belongs to `doc-ref` — it's possible to put non-URL references there, and nowadays it's possible to put multiple, i.e.

```
doc-ref:
  - https://www.eurocontrol.int/publication/cat034-eurocontrol-specification-surveillance-data-exchange-part-2b
  - SUR.ET1.ST05.2000-STD-02b-01, Edition Number: 1.27
```

Another thing you might want to consider is `meta/xref/...` and reserving special tag for these SUR references there.